x/bor/keeper/msg_server.go

```diff
@@ -259,6 +260,67 @@ func (s msgServer) BackfillSpans(ctx context.Context, msg *types.MsgBackfillSpan
 	return &types.MsgBackfillSpansResponse{}, nil
 }
 
+func (s msgServer) SetProducerDowntime(ctx context.Context, msg *types.MsgSetProducerDowntime) (*types.MsgSetProducerDowntimeResponse, error) {
+	var err error
```

Comment:
Shall we validate the `MsgSetProducerDowntime` first? It contains an address we can verify even before calling the `util.FormatAddress`